### ISSUE TITLE
Can't make any Origin.

### ISSUE THREAD

**Issue by IAm2TheDude** [apace100/origins-fabric#129]
I've spent a really long time on this and It's just not working. I know this might be easier on discord but I don't like discord and I'm unwilling and unable to use it. I'm doing everything in the YouTube tutorial right as far as I can tell but I can't seem to get an origin to show up on the selection screen. The game itself is running fine, I've only had a couple crashes when I messed with the files a bit much, but I can never get a basic origin to show up. The example origins worked fine, (I was using the aerum to test stuff if that helps.) But the second I tried to alter anything it stopped working. I can change the description in pack.mcmeta, but that's about as much as I've been able to change without it not showing in the selection screen anymore. 
![dadsfesgfull](https://user-images.githubusercontent.com/79490172/108807243-7f23ca80-7569-11eb-8a7c-660f414970fa.jpg) Here's a run through of what I've done, going left to right. Have I made a mistake?

**Comment by apace100** [comment]
Yes. All files and folders within a datapack (not the datapack folder itself) need to be completely lowercase. That means your `Beevian` folder should be `beevian`, and so does your origin file. Don't forget to then update the reference in your origin layer file as well.

For more immediate help with datapacks, I suggest joining the Origins Discord. There are more people there who can help out. :) I only check these issues here from time to time.

**Comment by IAm2TheDude** [comment]
Okay so I did that, and it's still not working. Again I cannot use discord, which is why I am here. I'm willing to wait longer if it's alright for me to ask you here. I don't want to bother you by asking questions that should be asked somewhere else though. So if it's alright for me to ask, is there any other mistake I made? 
![rbebe](https://user-images.githubusercontent.com/79490172/109074143-f7e76b80-76bc-11eb-9af5-b10db365f5f2.png)
